ui/helpers/utils/conversions.util.js

```diff
@@ -135,12 +135,13 @@ export function decGWEIToHexWEI(decGWEI) {
   });
 }
 
-export function hexWEIToDecGWEI(decGWEI) {
+export function hexWEIToDecGWEI(decGWEI, { numberOfDecimals } = {}) {
   return conversionUtil(decGWEI, {
     fromNumericBase: 'hex',
     toNumericBase: 'dec',
     fromDenomination: 'WEI',
     toDenomination: 'GWEI',
+    numberOfDecimals,
```

Comment:
I'm not seeing where this is used?

Reply:
I removed it, it was added by @NiranjanaBinoy  but I did more changes to code and this was eventually not used anywhere